### PR TITLE
4.14 ODF template release

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
lgtm

File: config-templates/redhat/odf-local/changelog.json [hunk 1]
```diff
@@ -1,4 +1,21 @@
...
+        "4.14": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "30 January 2024",
+                "cves": "",
+                "golang": "1.20.11",
+                "ubi": "8.9-1029",
+                "new-features": [
+                    "Release of ODF 4.14"
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: `Release of ODF 4.14` ---> `Initial Release`

File: config-templates/redhat/odf-local/4.14/storage-class.yaml [hunk 2]
```diff
@@ -0,0 +1,139 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: storage.k8s.io/v1
+    kind: StorageClass
+    metadata:
+      name: sat-ocs-cephrbd-gold
+      labels:
+        app.kubernetes.io/name: "redhat-odf-local"
+        app.kubernetes.io/version: "v4.14.0"
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.14/storage-class.yaml:10]: The image version in deployment file is `v4.14.1`. So do we want to keep version here as `v4.14.0` or `v4.14.1`?

File: config-templates/redhat/odf-remote/changelog.json [hunk 3]
```diff
@@ -1,4 +1,21 @@
...
+        "4.14": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "30 January 2024",
+                "cves": "",
+                "golang": "1.20.11",
+                "ubi": "8.9-1029",
+                "new-features": [
+                    "Release of ODF 4.14"
```
Comment (mssachan) [inline config-templates/redhat/odf-remote/changelog.json]: same here

File: config-templates/redhat/odf-local/changelog.json [hunk 4]
```diff
@@ -1,4 +1,21 @@
 [
+    {
+        "4.14": {
+            "status": "supported",
+            "latest-revision": {
+                "revision": "1",
+                "release-date": "30 January 2024",
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]: Considering the change in STAGE push plan to have on 29th Jan now, please change release date to `31 January 2024`

**Review by mssachan — APPROVED** [review]
lgtm

**Pull request merged**